src/svadu.adoc

```diff
@@ -0,0 +1,92 @@
+  resulting from an explicit store, must be exact (i.e., non-speculative), and
+  observed in program order by the local hart. When two-stage address
+  translation is active, updates of the D bit in G-stage PTEs may be performed
+  as a result of speculative updates of the A bit in VS-stage PTEs.            +
+                                                                               +
+  The PTE update must appear in the global memory order before the memory access
+  that caused the PTE update and before any subsequent explicit memory access to
+  that virtual page by the local hart. The ordering on loads and stores provided
+  by FENCE instructions and the acquire/release bits on atomic instructions also
+  orders the PTE updates associated with those loads and stores as observed by
+  remote harts.                                                                +
+                                                                               +
+  The PTE update is not required to be atomic with respect to the memory access
+  that caused the update and a trap may occur between the PTE update and the 
+  memory access that caused the PTE update. If a trap occurs then the A and/or D
+  bit may be updated but the memory access that caused the PTE update might not
+  occur. The hart must not perform the memory access that caused the PTE update
+  before the PTE update is globally visible.
+
+[NOTE]
+====
+The PTE updates due to memory accesses ordered-after a FENCE are not themselves
+ordered by the FENCE.
+
+Simpler implementations that cannot precisely order the PTE update before
+subsequent explicit memory accesses to the associated virtual page by the local
+hart may simply order the PTE update before all subsequent explicit memory
+accesses to any virtual page by the local hart.
+====
+
+Svadu extension requires the page tables to be located in memory with hardware
+page-table write access and _RsrvEventual_ PMA.
+
+<<<
+
+The translation of virtual addresses (or guest physical addresses) to physical
+(or guest physical) addresses is accomplished with the same algorithm as
+specified in the Supervisor-Level ISA extension (section "Virtual Address
+Translation Process") and as modified by the hypervisor extension (section
+"Two-stage Address Translation"), except that step 7 of the translation process,
+instead of causing a page-fault exception due to A and/or D bits being 0 when
+required to be 1, continues as follows:
+
+[start=7]
+. If `pte.a = 0`, or if the original memory access is a store and `pte.d = 0`:
+.. If a store to `pte` would violate a PMA or PMP check, raise an access-fault
+   exception corresponding to the original access type.
+.. Perform the following steps atomically:
+... Compare `pte` to the value of the PTE at address `a + va.vpn[i] × PTESIZE`.
+... If the values match, set `pte.a` to 1 and, if the original memory access is
+    a store, also set `pte.d` to 1.
+... If the comparison fails, return to step 2
+
```

Comment:
I already integrated Svadu into the priv spec.  The confusion probably arose because it is part of `supervsior.adoc` rather than its own file.  So I think we should just close this PR.

Reply:
@wmat Please revert the PR (see my comments above)